proxy/src/registry.rs

```diff
@@ -64,6 +66,15 @@ pub enum Message {
     },
 }
 
+/// Breakdown of costs of a [`Transaction`].
+#[derive(Clone)]
+pub struct TransactionCosts {
+    /// Deposit held for registrations.
+    pub deposit: u64,
+    /// Cost for the transaction to be processed and included.
```

Comment:
Given the nature of our fees, it's good to explain this more accordingly. Something like:

```suggestion
    /// Transaction fee defined by the user. The higher the fee, the higher the priority
    /// given to the transaction. It needs to be at least the minimum accepted by the registry.
```